data/json/monsters.json

```diff
@@ -2584,7 +2584,7 @@
     "id": "mon_yugg",
     "type": "MONSTER",
     "name": "yugg",
-    "description": "This is a huge, slimy worm-like creature.  Its pale, flattened head drips an oily mucus as it breaches the ground, searching for prey.  Its pinkish mouth opens and closes, revealing long fangs glistening with ropey strands of saliva, which leave smoldering stains wherever they drip.  The mere sight of its putrid whitish visage is enough to loose prehistoric terrors within the darkest recesses of your mind.",
+    "description": "A gigantic, slimy worm-like creature the size of a pickup truck.  Its pale, flattened head drips an oily mucus as it breaches the ground, searching for prey.  Its pinkish mouth opens and closes, revealing long fangs glistening with ropey strands of saliva, which leave smoldering stains wherever they drip.  A single sight at its visage recalls a terrible prehistoric terror, a gnawing sense of powerlessness against something far beyond your comprehension.",
```

Comment:
Rather than 'sight', the word 'look' is more natural here. 'Look' serves as a verb while 'sight' is a noun in the original sentence and so doesn't make sense to retain in the revised sentence.